### PR TITLE
SB-25552 - UCI modules configuration update

### PR DESCRIPTION
**Pull request by shiva-rakshith**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-25552" title="SB-25552" target="_blank">StorySB-25552</a>  UCI - Deployment
  </td></table>
  <br />
 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by shiva-rakshith** [comment]
@keshavprasadms Please review and approve the PR.

**Comment by keshavprasadms** [comment]
Changes to this file needs to be separate PR `ansible/roles/kong-api/defaults/main.yml`
Changes to this file needs to be separate PR `kubernetes/helm_charts/core/nginx-private-ingress/templates/configmap.yaml`

**Comment by shiva-rakshith** [comment]
@keshavprasadms Removed code changes related to two files.

File: ansible/roles/stack-sunbird/templates/inbound.env [hunk 1]
```diff
@@ -1,25 +1,38 @@
...
+#Cassandra
+CASSANDRA_URL={{sunbird_cassandra_host}}
+CASSANDRA_PORT=9042
+CASSANDRA_KEYSPACE={{env}}_uci_store
+
+#Ports
+INBOUND_INTERNAL_PORT=8085
+
+#ODK
+ODK_URL=https://agg.staging.saksham.samagra.io/
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/inbound.env:33]: Why is this pointing to samagra domin? Is this how it should be?

File: ansible/roles/stack-sunbird/templates/inbound.env [hunk 2]
```diff
@@ -1,25 +1,38 @@
...
+ODK_USER=test
+ODK_PASS=abcd1234
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/inbound.env:35]: No user and pass in public repos

Comment (keshavprasadms) [reply ansible/roles/stack-sunbird/templates/inbound.env:35]: Using the URL and the creds I am able to login to the system. If I can do things on the system, nobody will even know who did it

File: ansible/roles/stack-sunbird/templates/transformer.env [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+CASSANDRA_PORT=9042
+CASSANDRA_KEYSPACE={{env}}_uci_store
+
+#Ports
+TRANSFORMER_INTERNAL_PORT=9091
+
+#ODK
+ODK_URL="http://odk-service.{{namespace}}.svc.cluster.local:8080"
+ODK_USER=administrator
+ODK_PASS=administrator
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/transformer.env:35]: Same comment as above for user and pass

File: ansible/roles/stack-sunbird/templates/inbound.env (hunk 1 above)
Comment (shiva-rakshith) [reply ansible/roles/stack-sunbird/templates/inbound.env]: Changed the URL.

File: ansible/roles/stack-sunbird/templates/inbound.env (hunk 2 above)
Comment (shiva-rakshith) [reply ansible/roles/stack-sunbird/templates/inbound.env]: Can you please the following variables to private repo, I have made changes to use those variables here.
ODK_USER={{uci_odk_user}}
ODK_PASS={{uci_odk_password}}
uci_odk_user=administrator
uci_odk_password=administrator

File: ansible/roles/stack-sunbird/templates/transformer.env (hunk 3 above)
Comment (shiva-rakshith) [reply ansible/roles/stack-sunbird/templates/transformer.env]: Made changes to use variables.

**Review by keshavprasadms — APPROVED** [review]

File: ansible/roles/stack-sunbird/templates/inbound.env (hunk 2 above)
Comment (keshavprasadms) [reply ansible/roles/stack-sunbird/templates/inbound.env]: I have added these vars in private. Going forward you need to add the vars to dev env and test it out. Please get access for the same

Comment (shiva-rakshith) [reply ansible/roles/stack-sunbird/templates/inbound.env]: Sure, Thank you!

**Pull request merged**